geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/DestroyGatewaySenderCommandTest.java

```diff
@@ -88,7 +88,7 @@ public void oneFunctionReturnsError() throws Exception {
     functionResults.add(result2);
 
     doReturn(mock(Set.class)).when(command).getMembers(any(), any());
-    parser.executeAndAssertThat(command, "destroy gateway-sender --id=1").statusIsError()
+    parser.executeAndAssertThat(command, "destroy gateway-sender --id=1").statusIsSuccess()
```

Comment:
I'm confused as to why something displaying `ERROR: result2` is a successful execution.  I would be more inclined to say that any member error constitutes a command error, rather than any success constitutes command success.